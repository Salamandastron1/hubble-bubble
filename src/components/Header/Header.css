@import url('https://fonts.googleapis.com/css?family=Titan+One');
.begin-game {
    display: grid;
    justify-content: center;
    align-items: center;
    height: 60%;
    animation: spin-right 5s linear infinite;
}
.game-nav {
    font-family: 'Titan One', cursive;
    text-decoration: none;
    color: white;
    font-size: 2em;
    text-align: center;
    background: #c17cb3;
    padding: 10px;
    border-radius: 15px;
}

.game-nav:hover {
    font-family: 'Titan One', cursive;
    text-decoration: none;
    color: white;
    text-align: center;
    background: #97638c;
    padding: 10px;
    border-radius: 15px;
    box-shadow: inset -4px -3px 2px white
}
.planet-orbit {
    -webkit-clip-path: polygon(50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
    clip-path: polygon(50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
    background: #e5c5c4;
    height: 100%;
}
.button-text {
    animation: spin-left 5s linear infinite;
}
@-webkit-keyframes spin-right {

    100% {
        -webkit-transform: rotate(360deg);
    }
}
@-webkit-keyframes spin-left {
    0% {
        -webkit-transform: rotate(360deg);
    }
    100%{
        -webkit-transform: rotate(0deg);
    }
}

@media (min-width: 320px) and (max-width: 540px) {
    .begin-game {
        clip-path: polygon(50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
        -webkit-clip-path: polygon(50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
        height: 320px;
        width: 320px;
        display: grid;
        background: #c17cb3;
        margin: auto;
        margin-top: 100px;
    }
    .planet-orbit {
        display: none;
    }
    .game-nav {
        align-self: center;
        border: solid;
        font-size: 1.5em;
    }
}
@media (min-width: 540px) and (max-width: 739px) {
  .begin-game {
        clip-path: polygon(50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
        -webkit-clip-path: polygon(50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
        height: 320px;
        width: 320px;
        display: grid;
        background: #c17cb3;
        margin: auto;
        margin-top: 100px;
    }
    .planet-orbit {
        display: none;
    }
    .game-nav {
        align-self: center;
        border: solid;
        font-size: 1.5em;
    }  
}