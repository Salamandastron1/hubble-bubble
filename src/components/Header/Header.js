import React from 'react'
import { NavLink } from 'react-router-dom';
import './Header.css'
import satellite from '../../assets/Aha-Soft-Transport-First-satellite.ico'

const Header = (props) => {
  return (
    <section className='begin-game'>
    <img src={satellite}/>
      <NavLink 
        className='game-nav'
        to='/hubbleTrouble'>
        Start Stellar Star Stream 
      </NavLink>
    <img src={satellite} />
    </section>
  )
}

export default Header;