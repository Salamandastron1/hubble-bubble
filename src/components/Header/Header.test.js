import Header from './Header'
import React from 'react'
import { shallow } from 'enzyme'


it('should match the snapshot', () => {
  const wrapper = shallow(<Header />)

  expect(wrapper).toMatchSnapshot()
})