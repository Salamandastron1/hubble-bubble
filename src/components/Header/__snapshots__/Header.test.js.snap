// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<section
  className="begin-game"
>
  <div
    className="planet-orbit"
  />
  <div
    className="button-text"
  >
    <NavLink
      activeClassName="active"
      aria-current="page"
      className="game-nav"
      to="/gametime"
    >
      Start!!!
    </NavLink>
  </div>
  <div
    className="planet-orbit"
  />
</section>
`;
